frontend/src/bundles/password-reset/pages/password-reset.tsx

```diff
@@ -22,14 +22,17 @@ import {
     type PasswordResetPayload,
     type PasswordResetRequestDto,
 } from '~/bundles/password-reset/types/types.js';
+import { notificationManager } from '~/framework/notification/notification.js';
 
 import { PasswordResetSuccessMessage } from '../components/components.js';
 
+const errorText = 'The password reset link you used has expired';
```

Comment:
but let's move this to separate file as constant

Reply:
all done